### ISSUE TITLE
Home page looks broken after fresh install (messages and cards conflict)

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#6074]
## Description of the bug

After a fresh install of Backdrop, the new user is redirected to the home page, which looks pretty broken after the addition of our new cards display. 

The problem is that the messages appear between the hero and the cards, but the spacing around the messages is uneven, with huge whitespace on the top and sites, but no space on the bottom.

<img width="1451" alt="Screenshot 2023-04-20 at 2 21 18 PM" src="https://user-images.githubusercontent.com/397895/233490718-5ccd4c55-f27a-4ad9-a097-f4e6f38f14a1.png">

I recommend we adjust the spacing around the messages (when cards are present).

## Steps To Reproduce

To reproduce the behavior:
1. Install backdrop 

## Additional information

Add any other information that could help, such as:
- Backdrop CMS version:
- Web server and its version:
- PHP version: 
- Database sever (MySQL or MariaDB?) and its version:
- Operating System and its version:
- Browser(s) and their versions:

**Comment by laryn** [comment]
See also: https://github.com/backdrop/backdrop-issues/issues/5769

**Comment by klonos** [comment]
Yup. #5769 seems to be the issue we should be looking at for this problem. @jenlampton do you feel that this is a duplicate of that? ...is everything you have concerns about when it comes to this issue here already covered over in #5769, or are there distinctly different things that you were planning for this issue here?

**Comment by jenlampton** [comment]
Yes, this was 100% a duplicate. I knew someone must have made another issue, I was surprised when I couldn't find one! Search failed me. ty for closing.

**Comment by indigoxela** [comment]
> Yes, this was 100% a duplicate. I knew someone must have made another issue...

We should really fix the problem, so people stop reporting it. :wink: We can now with supplemental stylesheets.

The other issue https://github.com/backdrop/backdrop-issues/issues/5769 has more activity and is older, so closing this one here as duplicate.
